codalab/client/local_bundle_client.py

```diff
@@ -717,6 +717,15 @@ def get_worksheet_info(self, uuid, fetch_items=False, fetch_permission=True):
 
         if fetch_items:
             result['items'] = self._convert_items_from_db(result['items'])
+            # get a list of meta-data fields that require reformatting
+            metadata_reformat_field_list = [(name, func) for (_, name, func) in \
+                worksheet_util.get_default_schemas()['default'] if func]
+            metadata_list = [ additional_info.get('metadata') for (additional_info, _, _, _) in \
+                result['items'] if additional_info and additional_info.get('metadata')]
+            for metadata in metadata_list:
+                for (name, func) in metadata_reformat_field_list:
+                    if metadata.get(name):
+                        metadata[name] = worksheet_util.apply_func(func, metadata[name])
```

Comment:
We do not want to render the metadata here because get_worksheet_info should preserve the underlying information.  The formatting should be done in the codalab repo side.